challenge.js

```diff
@@ -26,6 +27,7 @@
  */
 
 // ANSWER
+E is a descendent
```

Comment:
Since this answer is so general, it's difficult to know how correct it is. E is a grandchild of A. 3/4